docs/develop/node/redis-om/index-redis-om.mdx

```diff
@@ -0,0 +1,619 @@
+
+Yep. Looks like JSON.
+
+### Add a Read Route
+
+Create down, let's add a GET route to read this song from HTTP instead of using the `redis-cli`:
+
+```javascript
+router.get('/:id', async (req, res) => {
+
+  // fetch the Song 
+  let song = await repository.fetch(req.params.id)
+
+  // return the Song we just fetched
+  res.send(song)
+
+})
+```
+
+Now you can use `curl` or your browser to load `http://localhost:8080/song/01FKRW9WMVXTGF71NBEM3EBRPY` to fetch the song:
+
+    $ curl -X GET http://localhost:8080/song/01FKRW9WMVXTGF71NBEM3EBRPY -s | json_pp
+
+And you should get back...
+
+```json
+{
+   "entityId" : "01FKRW9WMVXTGF71NBEM3EBRPY",
+   "entityData" : {
+      "artist" : "Dylan Beattie and the Linebreakers",
+      "title" : "HTML",
+      "lyrics" : "W3C, RFC, a JIRA ticket and a style guide.\\nI deploy with FTP, run it all on the client side\\nDon’t need Ruby, don’t need Rails,\\nAin’t nothing running on my stack\\nI’m hard wired, for web scale,\\nYeah, I’m gonna bring the 90s back\\n\\nI’m shipping HTML,\\nHTML,\\nI’m shipping HTML,\\nHTML…\\n\\nNo logins, no trackers,\\nNo cookie banners to ignore\\nI ain’t afraid of, no hackers\\nJust the occasional 404\\nThey hatin’, what I do,\\nBut that’s ‘cos they don’t understand\\nMark it up, break it down,\\nRemember to escape your ampersands…\\n\\nI’m shipping HTML,\\nHTML,\\nI’m shipping HTML,\\nHTML…\\n\\n(But it’s really just markdown.)",
+      "link" : "https://www.youtube.com/watch?v=woKUEIJkwxI",
+      "genres" : [
+         "blues rock",
+         "hard rock",
+         "parody",
+         "rock"
+      ],
+      "duration" : 220,
+      "music" : "\"Highway to Hell\" by AC/DC",
+      "year" : 2020
+   }
+}
+```
+
+...complete wrong results! What's up with that? Well, our entity doesn't know how to serialize itself to JSON so Express is doing that for us. And since `JSON.stringify`, which Express is using, can't see the properties that the `Schema` class added to our entity, it's just rendering the internal implementation of the entity. We can fix that by making our `Song` smarter.
```

Comment:
```suggestion
...completely wrong results! What's up with that? Well, our entity doesn't know how to serialize itself to JSON so Express is doing that for us. And since `JSON.stringify`, which Express is using, can't see the properties that the `Schema` class added to our entity, it's just rendering the internal implementation of the entity. We can fix that by making our `Song` smarter.
```